server/methods.js

```diff
@@ -0,0 +1,22 @@
+ Meteor.startup(function() {
+  if(Meteor.isServer) {
+    fuzzy = Npm.require("fuzzy");
+  }
+});
+
+ Meteor.methods({
+  findDreamCollegeId: function(userInputCollege) {
+    check(userInputCollege, String);
+    var colleges = CollegesInMemory;
+    var collegeNameArray = [];
+    var collegeNameHash = {}
+    _.each(colleges, function(college) {
+      collegeNameArray.push(college.name);
+      collegeNameHash[college.name] = college._id;
+    })
+    var results = fuzzy.filter(userInputCollege, collegeNameArray);
```

Comment:
`server/` `client/` and `lib/` folders aren't recognized in packages like they are in normal apps. In packages, it all depends on the architectures your specify in your `addFiles` call.